modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/loading/Load.scala

```diff
@@ -119,14 +120,9 @@ object Load {
       .limitRetries[LoaderAction[F, *]](MaxRetries)
       .join(RetryPolicies.exponentialBackoff(Backoff))
 
-  private def congratulate[F[_]: Monad: Logging: Timer](message: LoaderMessage.ShreddingComplete): LoaderAction[F, Unit] =
-    Timer[F].clock.instantNow.flatMap { now =>
-      val count = message.count.map(c => s"${c.good} events").getOrElse("volume is unknown")
-      val latency = message.timestamps.min match {
-        case Some(earliest) => s"${Duration.between(earliest, now).toSeconds} seconds"
-        case None => "unknown"
-      }
-      Logging[F].info(s"Folder [${message.base}] has been loaded and committed. Success!") *>
-        Logging[F].info(s"Folder [${message.base}] ($count) latency: $latency")
-    }.liftA
+  private def congratulate[F[_]: Logging: Monad](
+    loaded: LoaderMessage.ShreddingComplete
+  ): F[Unit] =
+    Logging[F].info(s"Folder ${loaded.base} loaded successfully") >>
+    Logging[F].sendMetrics(loaded)
```

Comment:
@chuwy can you please confirm that if something goes wrong inside `sendMetrics`, it won't affect the result of `load` because we are using `<*` in front of `congratulate` ?

Reply:
Yes, can confirm that. `<*` doesn't change order/semantics of execution, it just tells that RHS value will be ignored and LHS returned. Whereas `*>` does it vice versa - LHS ignored, RHS returned.

Reply:
That's what I had understood, perfect